pages/index.html

```diff
@@ -33,9 +33,9 @@
 	<div class="twelve columns">
 		<h1>Connect to #jquery-meeting</h1>
 
-		<p>The jQuery team hosts all of its meeting in the #jquery-meeting IRC channel on Freenode.net. Normally everyone in the channel is muted unless a meeting is going on. Please be respectful to those participating. You can connect to the channel using an IRC client or through the web form below:</p>
+		<p>The jQuery team hosts all of its meeting in the #jquery-meeting IRC channel on Freenode.net. Normally everyone in the channel is muted unless a meeting is going on. Please be respectful to those participating. You can connect to the channel using an IRC client or visit the link below:</p>
```

Comment:
"visit the link below" seems like a poor use of HTML. We should have something like `or use <a ...>webchat</a>`.